walkthroughs/week-2-web-development/examples/stanley/src/main/webapp/script.js

```diff
@@ -30,3 +30,7 @@ function randomizeImage() {
   imageContainer.innerHTML = '';
   imageContainer.appendChild(imgElement);
 }
+
+function init() {
+  document.getElementById("content").style.color = 'green';
```

Comment:
This is a great way of adding in some JS! One note though. What you have here is fine and should work as expected, but best practice tends to be adding or removing css classes on an element rather than tweaking the css styles individually. That way all of your styling information remains in the same place.

Reply:
Ohhh interesting! That makes sense, does that still happen through java script? Could you maybe give me a bit of an idea on where to start regarding that?

Reply:
Yep, that part still happens in JS. And certainly, take a look at `classList.add` and `classList.remove` here: https://developer.mozilla.org/en-US/docs/Web/API/Element/classList

Reply:
Thanks! Question, so how would I specifically change the font if my removing a whole class, wouldn't that affect the other properties the class has?

Reply:
Correct, if you remove a class then that will remove any properties it's affecting. The solution is to use multiple classes if you're only wanting to touch certain properties. Alternatively, you could also just add a class that overrides the properties you with to alter.